docs/installation.rst

```diff
@@ -340,44 +342,40 @@ current versions of Linux, macOS, and Windows.
 +----------------------------------+-------------+------------------------------+--------------------------------+-----------------------+
-| Gentoo Linux                     |Yes          | 2.7,3.2                      | 2.1.0                          |x86-64                 |
+| Gentoo Linux                     |Yes          | 2.7                          | 2.1.0                          |x86-64                 |
 +----------------------------------+-------------+------------------------------+--------------------------------+-----------------------+
 | Arch Linux                       |Yes          | 2.7,3.5                      | 3.4.1                          |x86,x86-64             |
 +----------------------------------+-------------+------------------------------+--------------------------------+-----------------------+
 | FreeBSD 10.2                     |Yes          | 2.7,3.4                      | 3.1.0                          |x86-64                 |
 +----------------------------------+-------------+------------------------------+--------------------------------+-----------------------+
-| Windows 7 Pro                    |Yes          | 2.7,3.2,3.3                  | 3.4.1                          |x86-64                 |
+| Windows 7 Pro                    |Yes          | 2.7,3.3                      | 3.4.1                          |x86-64                 |
 +----------------------------------+-------------+------------------------------+--------------------------------+-----------------------+
 | Windows Server 2008 R2 Enterprise|Yes          | 3.3                          |                                |x86-64                 |
 +----------------------------------+-------------+------------------------------+--------------------------------+-----------------------+
 | Windows Server 2012 R2           |Yes          | 2.7,3.3,3.4                  | 3.4.1 (CI target)              |x86,x86-64             |
 +----------------------------------+-------------+------------------------------+--------------------------------+-----------------------+
-| Windows 8 Pro                    |Yes          | 2.6,2.7,3.2,3.3,3.4a3        | 2.2.0                          |x86,x86-64             |
+| Windows 8 Pro                    |Yes          | 2.7,3.3,3.4a3                | 2.2.0                          |x86,x86-64             |
 +----------------------------------+-------------+------------------------------+--------------------------------+-----------------------+
-| Windows 8.1 Pro                  |Yes          | 2.6,2.7,3.2,3.3,3.4          | 2.4.0                          |x86,x86-64             |
+| Windows 8.1 Pro                  |Yes          | 2.7,3.3,3.4                  | 2.4.0                          |x86,x86-64             |
 +----------------------------------+-------------+------------------------------+--------------------------------+-----------------------+
```

Comment:
Might it still be useful leaving the historical version support here? If someone still uses Python 2.6 or 3.2, they can still see here what versions were good on what platforms. Of course, we won't bother testing 2.6 or 3.2 for future ones.